research-hub-web/src/app/app.component.ts

```diff
@@ -135,31 +230,33 @@ export class AppComponent implements OnInit, OnDestroy, AfterViewInit {
     this.appComponentService.setContentSidenavHasContent(hasContent);
   }
 
-  /**
-   * Sets the page title, and enables/disables the search bar and header.
-   * Also sets any custom CSS for the page.
-   * @param pageInfo currentPage information object.
-   */
-  setTitleSearchBarHeaderCustomCSS(pageInfo: any) {
-    if (pageInfo.title) {
-      this.titleService.setTitle('ResearchHub: ' + this.pageTitle);
-    }
-    this.headerService.setBatchParams(pageInfo.title, pageInfo.description, pageInfo.imageUrl, pageInfo.isHeaderVisible);
-    this.searchBarService.setVisibility(pageInfo.isSearchBarVisible);
-    this.appComponentService.setCustomCSSClassName(pageInfo.customCSSClassName);
-  }
+  async ngOnInit() {  
+    this.title = "Welcome to ResearchHub"
+    this.summary = "The ResearchHub connects you with people, resources, and services from across the University to enhance and accelerate your research."
```

Comment:
Finally 🥳 I always thought it was ridiculous that this stuff lived in a separate file 🤦